rskj-core/src/main/java/co/rsk/config/BridgeConstants.java

```diff
@@ -60,6 +60,8 @@
 
     protected int lockingCapIncrementsMultiplier;
 
+    protected long timeReceiveHeadersBetweenCalls;  // (seconds)
```

Comment:
```suggestion
    protected long minSecondsBetweenCallsToReceiveHeaders;  // (seconds)
```
Something more explanatory?